Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/cryptostore/StoreUtil.java

```diff
@@ -43,7 +44,7 @@
 public class StoreUtil {
 
     private static final Logger LOG = LoggerFactory.getLogger(StoreUtil.class);
-    public static final String INTERNAL_EXCHANGE = "INTERNAL";
+    public static final String INTERNAL_EXCHANGE = InternalExchangeManager.getInstance().getDefaultExchange();
```

Comment:
Rather than initialize this as a final static variable, Id rather call this method each time. If the internalExchangeInfo.xml file is updated, we lose the ability to reload the file and update this field.